backend/scripts/sync_prod_to_sqlite.py

```
@@ -0,0 +1,315 @@
+
+BACKEND_DIR = Path(__file__).resolve().parent.parent
+BACKUP_DIR = BACKEND_DIR / 'backups'
+# The snapshot is a full production fixture; it lives in the gitignored
+# backups/ so it can never be staged.
+SNAPSHOT = BACKUP_DIR / 'prod_snapshot.json'
+
+PROD_PARAM = '/tally/prod/database_url'
+PG_IMAGE = 'postgres:17'
+CONTAINER = 'tally-local-pg'
+PG_PORT = '5434'
+PG_PASSWORD = 'localpass'
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "PG_PASSWORD"

(S105)